llama_stack/apis/tools/__init__.py

```diff
@@ -0,0 +1,7 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the terms described in the LICENSE file in
+# the root directory of this source tree.
+
+from .tools import *  # noqa: F401 F403
```

Comment:
lets not add more imports that require noqa. We want to actually get rid of noqa in the full codebase

Reply:
@raghotham this is actually one of the few places where it is justified